packages/fe-fpm-writer/templates/filter/Controller.js

```diff
@@ -1,7 +1,7 @@
 sap.ui.define(["sap/ui/model/Filter", "sap/ui/model/FilterOperator"], function(Filter, FilterOperator) {
     "use strict";
     return {
-        <%- eventHandlerFnName %>: function(sValue) {
+        <%- (typeof eventHandlerFnName !== 'undefined' && eventHandlerFnName) || 'filterItems' %>: function(sValue) {
```

Comment:
Why is this one back? Aren't you still ensuring in your code that `eventHandlerFnName` is always defined?

Reply:
There are 2 cases where there is no `eventHandlerFnName` set: 
- eventHandler is not set/undefined, then event handler configuration is not applied,
- eventHandler is a string, then it is used as provided to link an existing event handler.

Reply:
Fixed with #1027